install_config/install/advanced_install.adoc

```diff
@@ -798,17 +798,56 @@ openshift_examples_modify_imagestreams=true
 |Set to `true` if pointing to a registry other than the default. Modifies the image stream location to the value of `*oreg_url*`.
 
 |`*openshift_docker_additional_registries*`
-|Specify the additional registry or registries.
-If the registry required to access the registry is other than `80` include the port number required.
+|Specify the additional registry or registries. If the registry required to access the registry is other than `80` include the port number required.
+
+|`*openshift_cockpit_deployer_prefix*`
+|Specify the URL and path to namespace where the registry-console image is located.
+
+|`*openshift_web_console_prefix*`
+|Specify the prefix for the web console images.
+
+|`*openshift_service_catalog_image_prefix*`
+|Specify the prefix for the service catalog component image.
+
+|`*ansible_service_broker_image_prefix*`
+|Specify the prefix for the ansible service broker component image.
+
+|`*template_service_broker_prefix*`
+|Specify the prefix for the template service broker component image.
+
+|`*openshift_crio_systemcontainer_image_override*`
+|A setting for if you are using CRI-O and if you are using an alternative CRI-O
+system container image from another registry.
 |===
 
 For example:
 ----
 oreg_url=example.com/openshift3/ose-${component}:${version}
 openshift_examples_modify_imagestreams=true
 openshift_docker_additional_registries=example.com:443
+openshift_cockpit_deployer_prefix='registry.example.com/openshift3/'
+ifdef::openshift-origin[]
+openshift_web_console_prefix='registry.example.com/openshift/origin-
+endif::[]
+ifdef::openshift-enterprise[]
+openshift_web_console_prefix='registry.example.com/openshift3/ose-
+endif::[]
+openshift_service_catalog_image_prefix='registry.example.com/openshift3/ose-'
+ansible_service_broker_image_prefix='registry.example.com/openshift3/ose-'
+ifdef::openshift-origin[]
+template_service_broker_prefix="registry.example.com/origin-"
+endif::[]
+ifdef::openshift-enterprise[]
+template_service_broker_prefix='registry.example.com/openshift3/ose-'
+endif::[]
```

Comment:
If possible, here is only need keep one pair of origin or enterprise version judgment, and define most of prefix there, just like this:
ifdef::openshift-origin[]
openshift_web_console_prefix='registry.example.com/openshift/origin-
openshift_service_catalog_image_prefix='registry.example.com/openshift/origin-'
ansible_service_broker_image_prefix='registry.example.com/openshift/origin-'
template_service_broker_prefix="registry.example.com/openshift/origin-"
endif::[]
ifdef::openshift-enterprise[]
openshift_web_console_prefix='registry.example.com/openshift3/ose-
openshift_service_catalog_image_prefix='registry.example.com/openshift3/ose-'
ansible_service_broker_image_prefix='registry.example.com/openshift3/ose-'
template_service_broker_prefix='registry.example.com/openshift3/ose-'
endif::[]